scripts/verify_clients_untracked_files.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
It is easier to reason about
```suggestion
#!/bin/sh
```
so if that also works, let's use it!

Reply:
if we work with `sh` we can't set `-o pipefail`:
```
scripts/verify_clients_untracked_files.sh: 3: set: Illegal option -o pipefail
```
and it's very unlikely that the github runner machines wont have bash in them, so I'm leaving it `bin/bash` :slightly_smiling_face: